pages/[...slug].vue

```diff
@@ -18,6 +18,35 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav v-if="breadcrumbs" class="mb-4">
+          <span class="flex-row">
+            <NuxtLink
+              class="border-2 border-solid border-gray-600 hover:border-accent-600"
+              :to="'/'"
+            >
+              <Icon name="heroicons:home-20-solid" />
+            </NuxtLink>
+            <Icon name="heroicons:chevron-right" class="mx-2" />
+          </span>
+
+          <span
+            v-for="(crumb, index) in breadcrumbs"
+            :key="crumb._path"
+            class="flex-row"
+          >
+            <template v-if="index < breadcrumbs.length - 1">
+              <NuxtLink class="underline text-accent-600" :to="crumb._path">
+                {{ crumb.title }}
+              </NuxtLink>
+              <Icon name="heroicons:chevron-right" class="mx-2" />
+            </template>
+            <span v-else>
+              {{ crumb.title }}
+            </span>
+          </span>
+        </nav>
```

Comment:
Before:
![image](https://github.com/exactpro/docs-toolkit/assets/51794590/99231624-5a5a-455d-a6c6-a3fc8abbe5ce)

After:
![image](https://github.com/exactpro/docs-toolkit/assets/51794590/b14540d8-2ce7-4aec-876c-e13a69fd5f83)

- Made home icon slightly bigger
- Make borders rounded
- `inline-flex` class removes unnecessary space on top of home button
- with `transition-colors` home button will change color gradientially
- using template elements made breadcrumbs elements siblings
- `nav` is main breadcrumb container now
- `flex-wrap` will wrap breadcrumbs if it is too big

```suggestion
        <nav v-if="breadcrumbs" class="mb-4 flex flex-wrap items-center gap-x-2">
          <NuxtLink
            class="border border-solid rounded text-gray-600 border-gray-600 hover:border-accent-600 hover:text-accent-600 transition-colors p-1 inline-flex"
            :to="'/'"
          >
            <Icon name="heroicons:home-20-solid" size="1.2em" />
          </NuxtLink>
          <Icon name="heroicons:chevron-right" />

          <template
            v-for="(crumb, index) in breadcrumbs"
            :key="crumb._path"
          >
            <template v-if="index < breadcrumbs.length - 1">
              <NuxtLink class="underline text-accent-600" :to="crumb._path">
                {{ crumb.title }}
              </NuxtLink>
              <Icon name="heroicons:chevron-right" />
            </template>
            <span v-else>
              {{ crumb.title }}
            </span>
          </template>
        </nav>
```